templates/bundles/PrestaShopBundle/Admin/Configure/AdvancedParameters/Webservice/index.html.twig

```diff
@@ -52,6 +52,45 @@
     {% endif %}
   {% endblock %}
 
+  {% block webservice_api_status %}
+    {% set devdocUrl = 'https://devdocs.prestashop.com/1.7/development/webservice/' %}
+
+    {% if webserviceStatus['enabled'] == true %}
+      <div class="card">
+        <h3 class="card-header">
+          <i class="material-icons">info_outline</i> {{ 'Webservice status'|trans }}
+        </h3>
+        <div class="card-block">
+          <div class="card-text">
+            <p>
+              {{ 'Webservice is enabled. Main entry point is' |trans }}
+              <a href="{{ webserviceStatus['endpoint'] }}">
+                {{ webserviceStatus['endpoint'] }}
+              </a>
+            </p>
+            {% if webserviceStatus['isFunctional'] == true %}
+              <p>
+                {{ "Read the [1]developer documentation[/1] to learn more about it."|trans({
+                  '[1]': '<a href="'~devdocUrl~'">', '[/1]': "</a>"
+                }, 'Admin.Advparameters.Feature')|raw }}
+              </p>
+            {% else %}
+              <p>
+                {{ 'It seems that the webservice endpoint is not functional. If you are using httpd/apache2, you need to enable URL rewriting on your server.'|trans }}
```

Comment:
Isn't it an error message? Could you tell me where it appears on the page please? :)

Reply:
Have a look at the screenshot 😉  https://user-images.githubusercontent.com/1533248/131145979-50dac2cd-5019-4eb7-a2ea-a044d8d72e5c.png

Reply:
Sorry, I didn't pay attention to the screenshots! 🙈